### DISCUSSION TITLE
How do I pass a variable from JS  into a Form?

### DISCUSSION THREAD

**Discussion by fmen1** [AutomaApp/automa#1509] [Q&A]
This is a JS snippet that extracts today's date into the variable "today"

//-------------------------------------------------------------------------------
var today = new Date();
var dd = String(today.getDate()).padStart(2, '0');
var mm = String(today.getMonth() + 1).padStart(2, '0'); //January is 0!
var yyyy = today.getFullYear();
today = yyyy + '-' + mm + '-' + dd ;
//--------------------------------------------------------------------------------

I've tried passing the variable into the Forms Text Field as {{variables.today}} and simply as today, without success.

Thanks

**Comment by kolampixel** [comment]
you can use `automaSetVariables('today', today)`
then use `{{ variables@today }}` ini text field

**Comment by fmen1** [comment]
I can't thank you enough for responding.  But, please, where do I add automaSetVariables('today', today)? I tried placing it in the JS script without success.
